src/psyclone/psyGen.py

```diff
@@ -4506,11 +4523,19 @@ def nodes_to_code_block(parent, statements):
         del statements[:]
         return code_block
 
+    def _create_schedule(self, name):
```

Comment:
pylint complains that this method does not use `self` - could make it static?

Reply:
Also, since we're here, could you formally document the :returns: and :rtype:.

Reply:
Converted to staticmethod and added doctring